website/www/site/content/en/get-started/quickstart-py.md

```diff
@@ -15,7 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Apache Beam Python SDK Quickstart
+# WordCount quickstart for Python
```

Comment:
Should we change the page title above to match this header?

Reply:
a friendly ping @VeronicaWasson @tvalentyn - would it be possible to fix this?

Reply:
Created this PR to fix this: https://github.com/apache/beam/pull/25471

Reply:
oh sorry, I missed this comment. thanks for fixing!